examples/todos-flow/components/Link.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from 'react'
+
+export type Props = {
+  active: boolean,
+  children?: React$Element<any>,
```

Comment:
According to http://www.saltycrane.com/blog/2016/06/flow-type-cheat-sheet/, React$* types are private.

It's not better like this ?
```
import type { Element } from 'react';
// ...
children?: Element<any>,
// ...
```
?